pom.xml

```diff
@@ -149,7 +149,7 @@
         <spring.version>4.3.10.RELEASE</spring.version>
         <jetty.version>9.3.24.v20180605</jetty.version>
         <pax.cdi.version>0.12.0</pax.cdi.version>
-        <pax.web.version>6.0.9</pax.web.version>
+        <pax.web.version>6.0.11</pax.web.version>
```

Comment:
✏️ Should the pax web version be tied with the jetty version? Would it be worthwhile to add a comment here?